assets/services.json

```diff
@@ -24,7 +24,11 @@
       "rules": [
         "||activisionblizzard.com^",
         "||activision.com^",
-        "||demonware.net^"
+        "||demonware.net^",
+        "||callofduty.com^",
+        "||callofdutyleague.com^",
+        "||codm.activision.com^",
+        "||codmwest.com^"
```

Comment:
Last two domains are not available. Are they needed?

Reply:
Check:
https://static.codmwest.com/seasonalvideo/index.html